tc39_test.go

```diff
@@ -696,7 +697,7 @@ func TestTC39(t *testing.T) {
 	// Tests ignored: 10,453, passed: 14,782
 
 	// at ddfe24afe3043388827aa220ef623b8540958bbd
-	// Tests ignored: 19,183, passed: 14,512
+	// Tests ignored: 19,164, passed: 14,529
```

Comment:
I grepped for `SKIP` and `PASS` ... but reduced the PASS count by 3 to remove the lines that weren't actually tc39 tests, but part of the output of golang's tests.

But I was getting FAILed tests for the other commit so decided to not update it in case you are doing something for that.

Reply:
It's just for information, don't worry about it, I'll remove it in the future.